charts/lab-partner/templates/ingress.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: lab-partner
+  labels:
+    {{- include "lab-partner.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
```

Comment:
looks like there isn't any room here to specify a host, which we'd need to fully configure the ingress.